execution.py

```diff
@@ -116,53 +234,143 @@ def format_value(x):
     else:
         return str(x)
 
-def recursive_execute(server, prompt, outputs, current_item, extra_data, executed, prompt_id, outputs_ui, object_storage):
```

Comment:
I realized I never responded to this comment. I attempted to refactor this, but enough data has to be passed around between all components that I'm not convinced it's actually more readable/maintainable.

Any opinion? https://github.com/guill/ComfyUI/commit/2ff98c09a7547e6694f1280a3f5295c99c55a9a7

Reply:
I think simply splitting this into smaller functions has merit of its own, even if there is lots of data passed between components - otherwise we're breaking the single responsibility principle. However, this is a matter of taste, not function, so feel free to ignore it if you don't agree.